flybywiresim/a32nx
# revert: visible pilot and co-pilot model in cockpit

This reverts commit 32f0fbe89f5aef58b345ef3e766f4681c6f67e52.






Closes #3482 

## Summary of Changes


## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


- Make sure MCDU options menu works properly
- Check that there are no pilots visible in the cockpit
- Check to make sure that nothing else is broken


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
